lib/data_hygiene/document_status_checker.rb

```diff
@@ -0,0 +1,40 @@
+class DataHygiene::DocumentStatusChecker
+  def initialize(document)
+    @document = document
+  end
+
+  def content_store?
+    routes.each do |route|
+      content_item = GdsApi.content_store.content_item(route[:path])
+      updated_at = DateTime.parse(content_item["updated_at"])
+      return false unless updated_at > edition.published_at # content-store must be later due to latency
```

Comment:
What if it makes it there in under a second?  Should this be `>=` instead?

Reply:
I figured `>` would check milliseconds, but on second thought I think it's better to use `>=`.